scripts/run-integration-tests.sh

```diff
@@ -26,6 +27,7 @@ ARCH=$(go env GOARCH)
 : "${RUN_CONFORMANCE:=false}"
 : "${RUN_TESTER_LB_ADDONS:=false}"
 : "${RUN_KOPS_TEST:=false}"
+: "${RUN_DEFAULT_CANARY:=false}"
```

Comment:
Can we set this to `true` by default and only skip for the weekly tests? 
Also, might want to rename to like `RUN_INTEGRATION_DEFAULT_CNI` as the plan is to run all integration tests in the future, not just canary tests.

Reply:
Done. I noticed that for the weekly job, since we call the script 4 times, we will run integration tests against latest CR four times. We can definitely improve this in the future.